src/libs/humanizer/utils.ts

```diff
@@ -121,50 +112,8 @@ export async function getNativePrice(network: Network, fetch: Fetch): Promise<nu
   return response[platformId].usd
 }
 
-// @TODO this should be moved outside of the humanizer as we should not use it in the humanizer
-export async function getTokenInfo(
-  humanizerSettings: HumanizerSettings,
-  address: string,
-  options: any
-): Promise<HumanizerFragment | null> {
-  let platformId = options?.network?.platformId
-  if (!platformId) {
-    options.emitError({
-      message: 'getTokenInfo: could not find platform id for token info api',
-      error: new Error(
-        `could not find platform id for token info api ${humanizerSettings.networkId}`
-      ),
-      level: 'silent'
-    })
-    platformId = humanizerSettings.networkId
-  }
-
-  try {
-    const queryUrl = `${baseUrlCena}/coins/${platformId}/contract/${address}`
-    let response = await options.fetch(queryUrl)
-    response = await response.json()
-    if (response.symbol && response.detail_platforms?.[platformId]?.decimal_place)
-      return {
-        type: 'token',
-        key: address.toLowerCase(),
-        value: {
-          symbol: response.symbol.toUpperCase(),
-          decimals: response.detail_platforms?.[platformId]?.decimal_place
-        },
-        isGlobal: true
-      }
-    return null
-  } catch (e: any) {
-    return null
-  }
-}
-
-export function checkIfUnknownAction(v: Array<HumanizerVisualization>): boolean {
-  try {
-    return !!(v[0].type === 'action' && v?.[0]?.content?.startsWith('Unknown action'))
-  } catch (e) {
-    return false
-  }
+export function checkIfUnknownAction(v: HumanizerVisualization[] | undefined): boolean {
```

Comment:
isnt it better with the try, catch, like before? Just in case

Reply:
not really. the `?` is sufficient to cover all potential edge cases here.
The only questionable thing is if
`v[0].content.startsWith` is not a function, but this is not really possible, because HumanizerVisualization `action` type should always have a `.content: string` property